src/components/app.tsx

```diff
@@ -163,13 +198,18 @@ export const App: React.FC<AppProps> = ({ className, model }) => {
                 <div className={classes.source}>
                     <h2 className={classes.editorTitle}>Source:</h2>
                     <div>
-                        <ControlledEditor
-                            value={files[selected]}
-                            onChange={(_, value) => updateSelected(value)}
-                            language={selected.endsWith('.st.css') ? 'css' : 'javascript'}
-                            options={{ minimap: { enabled: false }, scrollBeyondLastLine: false }}
-                            editorDidMount={resizeEditor}
-                        />
+                        {model.monacoUp && (
```

Comment:
why do we need monacoUp ? isn't it just works after registration? and if not what makes the app re-render after it's initialized?

Reply:
still have buggy behavior here not working as intended yet